### PR TITLE
docs(clientLogLevel): update clientLogLevel options

### PR DESCRIPTION
**Pull request by gabemeola**
From https://github.com/webpack/webpack-dev-server/pull/1825

* Changed `none` to `silent`
* Changed `warning` to `warn`
* Added `trace` and `debug

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/writers-guide/

### CODE REVIEW

**Comment by EugeneHlushko** [comment]
Hi @gabemeola 
You changes to dev server haven't been release yet.

Please ping me when its getting released.

**Comment by EugeneHlushko** [comment]
@evilebottnawi maybe worth doing a release?

**Comment by evilebottnawi** [comment]
Yes, in near future, we need revert one of commits

/cc @hiroppy can you take care about revert, no access to compute right now

**Comment by hiroppy** [comment]
@evilebottnawi 

> we need revert one of commits

Which the commit?

**Comment by evilebottnawi** [comment]
@hiroppy https://github.com/webpack/webpack-dev-server/commit/2f7f05276e10a2ce408e0903982b8bbe07699182

https://github.com/webpack/webpack-dev-server/issues/1777

It is invalid solution, now we support `sockPort` option for this case

**Comment by EugeneHlushko** [comment]
Hi @gabemeola 
No need to re-merge master time to time, we will do it before merge

**Comment by hiroppy** [comment]
@EugeneHlushko I want to merge this pr. Related https://github.com/webpack/webpack-dev-server/pull/2021
And I'll delete `none` and `warning` if the major version is released.

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -175,27 +175,29 @@ webpack-dev-server --bonjour
 
 ## `devServer.clientLogLevel`
 
-`string: 'none' | 'info' | 'error' | 'warning'`
+`string: 'silent' | 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'none' | 'warning'`
```
Comment (hiroppy) [inline src/content/configuration/dev-server.md]: related https://github.com/webpack/webpack-dev-server/blob/master/client-src/default/utils/log.js#L5-L15

**Comment by EugeneHlushko** [comment]
okay lets see tests pass and we merge this

**Comment by EugeneHlushko** [comment]
Thanks! CC @hiroppy

**Comment by hiroppy** [comment]
Thanks!

**Pull request merged**